src/models/converters/data_decoded.rs

```diff
@@ -0,0 +1,58 @@
+use crate::models::commons::DataDecoded;
+use crate::models::service::transactions::SettingsInfo;
+
+impl DataDecoded {
+    pub(super) fn to_settings_info(&self) -> SettingsInfo {
```

Comment:
Should we return a result here? If any of the `unwrap` fail thenwe cannot render any tx for this safe, this seems a little risky. I would write it in a way that in case we cannot handle it or fail when trying to handle it we return our "Unknown" or `null`